src/main/java/me/totalfreedom/totalfreedommod/AntiNuke.java

```diff
@@ -35,10 +35,10 @@ public void onBlockBreak(BlockBreakEvent event)
 
         if (fPlayer.incrementAndGetBlockDestroyCount() > ConfigEntry.NUKE_MONITOR_COUNT_BREAK.getInteger())
         {
-            FUtil.bcastMsg(player.getName() + " is breaking blocks too fast!", ChatColor.RED);
-            //plugin.ae.autoEject(player, "You are breaking blocks too fast. Nukers are not permitted on this server.");
-            player.kickPlayer(ChatColor.RED + "You are breaking blocks too fast. Nukers are not permitted on this server.");
-
+            server.broadcast(Component.text(player.getName()).append(Component.text(" is breaking blocks too fast!"))
+                    .color(NamedTextColor.RED));
```

Comment:
Let's just modify the actual functions in FUtil to call Components instead of Strings.

Reply:
Why? I don't see the point in using FUtil.bcastMsg when it's literally just an inferior version of Server.broadcast.

Reply:
> Why? I don't see the point in using FUtil.bcastMsg when it's literally just an inferior version of Server.broadcast.

while you're at it also deprecate the FLog class and all of its methods and redirect all of the logging calls to the plugin's SLF4J logger. not really much reason for the FLog class to exist imho